### ISSUE TITLE
Remove quantity field from all EconomicResource forms

### ISSUE THREAD

**Issue by bhaugen** [FreedomCoop/valuenetwork#258]
The quantity field in EconomicResource is not editable:
https://github.com/FreedomCoop/valuenetwork/blob/master/valuenetwork/valueaccounting/models.py#L4373

That was changed awhile back because accounting rules say that resource quantities should only be changed by EconomicEvents. At that same time, we added an Adjust Quantity button on the resource event history page so that (for example) beginning balances or recounts could be handled and an event would be added to the resource history.

But we were inconsistent in that set of changes. Some forms still include the quantity field, and this is causing an error message in Django 1,10, which is fussier about these inconsistencies.

Some examples:

The CreateEconomicResourceForm
https://github.com/FreedomCoop/valuenetwork/blob/master/valuenetwork/valueaccounting/forms.py#L337 
has an editable quantity field.
When I go to an existing resource, for example
https://testocp.freedomcoop.eu/accounting/resource/26/
the change form does not have a quantity.
If I go to the event history, 
https://testocp.freedomcoop.eu/accounting/event-history/26/
it has an Adjust Quantity button.
But if I go to a resource type page, for example
https://testocp.freedomcoop.eu/accounting/resource-type/31/
and click the Create Resource button, the form does offer a quantity, but it doesn't get saved.
https://testocp.freedomcoop.eu/accounting/resource/106/

Basically, the quantity field should be removed from all EconomicResource forms (where model = EconomicResource).

**Comment by XaviP** [comment]
Yes, the error that django 1.10 gives:
```
django.core.exceptions.FieldError: 'quantity' cannot be specified for EconomicResource model form as it is a non-editable field.
```
 is only the [first ocurrence in forms.py](https://github.com/FreedomCoop/valuenetwork/blob/master/valuenetwork/valueaccounting/forms.py#L250) that it finds, but there are a lot of them.

**Comment by bhaugen** [comment]
> there are a lot of them.

I only found 3 where model = EconomicResource and only 2 that included the quantity field.

**Comment by XaviP** [comment]
Ok then, I just did a fast&dirty research.
